github/CheckRun.py

```diff
@@ -0,0 +1,326 @@
+# -*- coding: utf-8 -*-
+
+############################ Copyrights and license ############################
+#                                                                              #
+# Copyright 2020 Dhruv Manilawala <dhruvmanila@gmail.com>                      #
+#                                                                              #
+# This file is part of PyGithub.                                               #
+# http://pygithub.readthedocs.io/                                              #
+#                                                                              #
+# PyGithub is free software: you can redistribute it and/or modify it under    #
+# the terms of the GNU Lesser General Public License as published by the Free  #
+# Software Foundation, either version 3 of the License, or (at your option)    #
+# any later version.                                                           #
+#                                                                              #
+# PyGithub is distributed in the hope that it will be useful, but WITHOUT ANY  #
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS    #
+# FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more #
+# details.                                                                     #
+#                                                                              #
+# You should have received a copy of the GNU Lesser General Public License     #
+# along with PyGithub. If not, see <http://www.gnu.org/licenses/>.             #
+#                                                                              #
+################################################################################
+
+import datetime
+
+import github.CheckRunAnnotation
+import github.CheckRunOutput
+import github.GithubApp
+import github.GithubObject
+import github.PaginatedList
+import github.PullRequest
+
+
+class CheckRun(github.GithubObject.CompletableGithubObject):
+    """
+    This class represents check runs.
+    The reference can be found here https://docs.github.com/en/rest/reference/checks#check-runs
+    """
+
+    def __repr__(self):
+        return self.get__repr__(
+            {"id": self._id.value, "conclusion": self._conclusion.value}
+        )
+
+    @property
+    def app(self):
+        """
+        :rtype: :class:`github.GithubApp.GithubApp`
+        """
+        self._completeIfNotSet(self._app)
+        return self._app.value
+
+    @property
+    def check_suite(self):
+        """
+        :rtype: dict
```

Comment:
Surely this is a type of CheckSuite? May require another PR to be merged first.

Reply:
I thought the same but then I looked at the payload and we are just getting a dictionary with one attribute in it: `id` of the CheckSuite.
```
...
},
  "name": "mighty_readme",
  "check_suite": {
    "id": 5
  },
  "app": {
  ...
```
Reference: https://docs.github.com/en/free-pro-team@latest/rest/reference/checks#get-a-check-run

Reply:
Then we should probably unwrap it and call the property check_suite_id with type int

Reply:
Done 👍